plugins/python-build/bin/pyenv-uninstall

```diff
@@ -59,29 +52,43 @@ IFS=$'\n' scripts=(`pyenv-hooks uninstall`)
 IFS="$OLDIFS"
 for script in "${scripts[@]}"; do source "$script"; done
 
+uninstall-python() {
+  local DEFINITION="$1"
+  case "$DEFINITION" in
+  "" | -* )
+    usage 1 >&2
+    ;;
+  esac
```

Comment:
Shouldn't we validate the command line before starting any work?

Reply:
I agree, we should. will fix this in a subsequent commit. Also, do you think it would be a good idea to do the directory check `[ ! -d $PREFIX]` for all versions beforehand too? (when not running in --force mode). This would make the command behavior more atomic. but on the other hand, I don't see much issue with the uninstall command not being atomic in interactive mode.